unirec/unirec.h

```diff
@@ -764,6 +764,17 @@ ur_template_t *ur_create_template_from_ifc_spec(const char *ifc_data_fmt);
  */
 void ur_free_template(ur_template_t *tmplt);
 
+/** \brief Compares fields of two UniRec templates
+ * Function compares only sets of UniRec fields (direction is not compared).
+ * \note As it holds for UniRec templates in general, function returns invalid results if some
```

Comment:
This \note is note clear to me. `ur_template_compare()` doesn't return any "invalit results" value, it returns just 0 or 1...

Reply:
Ok, I understand now. So there should be a little bit different note, something like this:
```
This function does not check if arguments are valid UniRec templates. Caller must check before calling this function. Return value is undefined when any argument IS NOT a valid UniRec template.
```

Reply:
Ok, I will update the note if it is clearer this way.